### ISSUE TITLE
usePublications should have restrictEventTypesTo in metadataFilter

### ISSUE THREAD

**Issue by jsonpreet** [lens-protocol/lens-sdk#413]
Hi, I want to fetch publications based on their type: Mirror and Comment separately, but there's no option to add publication type in the request.

So I request you to please add restrictEventTypesTo in the usePublications request.

**Comment by moesalih** [comment]
hi @cesarenaldi . This one still doesn't seem fixed in 1.2.2. Do you know when it will be published?

**Comment by krzysu** [comment]
@moesalih it is available for now in the `next` release and will be available in 1.3.0 soon.
You use it as `publicationTypes` request prop, see the example 

```ts
const { data, loading, error } = usePublications({
  profileId: profileId('0x0635')
  publicationTypes: [ PublicationTypes.Post ]
});
```

to install next release use

```bash
npm install @lens-protocol/react@next
# or
npm install @lens-protocol/react-web@next
```
